translate/storage/test_mozilla_lang.py

```diff
@@ -121,3 +127,59 @@ def test_ok_translations(self, ok, target, istranslated):
         assert unit.source == "Source"
         assert unit.target == target
         assert unit.istranslated() == istranslated
+
+    def test_headers(self):
+        """Ensure we can handle and preserve file headers"""
+        lang = ("## active ##\n"
+                "## some_tag ##\n"
+                "## another_tag ##\n"
+                "## NOTE: foo\n"
+                ";Source\n"
+                "Target\n"
+                "\n\n")
+        store = self.StoreClass.parsestring(lang)
+        assert (
+            store.getlangheaders()
+            == [u'## some_tag ##',
+                u'## another_tag ##',
+                u'## NOTE: foo'])
+        out = io.BytesIO()
+        store.serialize(out)
+        out.seek(0)
+        assert (
+            out.read()
+            == ("## active ##\n"
+                "## some_tag ##\n"
+                "## another_tag ##\n"
+                "## NOTE: foo\n"
+                "\n\n"
+                ";Source\n"
+                "Target\n"
+                "\n\n"))
+
+    def test_not_headers(self):
+        """Ensure we can handle and preserve file headers"""
+        lang = ("## active ##\n"
+                "## some_tag ##\n"
+                "## another_tag ##\n"
+                "## NOTE: foo\n"
+                "## TAG: fooled_you ##\n"
+                ";Source\n"
+                "Target\n"
+                "\n\n")
+        store = self.StoreClass.parsestring(lang)
+        assert "## TAG: fooled_you ##" not in store.getlangheaders()
+
+    def test_tag_comments(self):
+        """Ensure we can handle and preserve miltiline comments"""
+        lang = ("## active ##\n"
+                "# First comment\n"
+                "## TAG: important_tag\n"
+                "# Second comment\n"
+                "# Third comment\n"
+                "## TAG: another_important_tag\n"
+                ";Source\n"
+                "Target\n"
+                "\n\n")
```

Comment:
personally i really dislike triple quotes in methods etc- but can we focus on the important things please